include/seqan3/core/detail/strong_type.hpp

```diff
@@ -59,6 +59,8 @@ enum struct strong_type_skill
 
 namespace seqan3
 {
+//!\implements enum_bitwise_operators
+//!\sa seqan3::add_enum_bitwise_operators<seqan3::detail::trace_directions> enables combining enum values.
 //!\cond
 template <>
 constexpr bool add_enum_bitwise_operators<seqan3::detail::strong_type_skill> = true;
```

Comment:
Since there is `\cond` this documentation should have no effect.

Reply:
That is true, so should I remove the `\cond`?